src/credentials/PersistentStorageKey.h

```diff
@@ -0,0 +1,61 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app/util/basic-types.h>
+#include <string.h>
+
+namespace chip {
+namespace Credentials {
+
+class PersistentStorageKey
+{
+public:
+    PersistentStorageKey() = default;
+
+    const char * FabricTable(chip::FabricIndex fabric) { return Format("f/%u/t", fabric); }
+    const char * FabricGroups(chip::FabricIndex fabric) { return Format("f/%u/g", fabric); }
```

Comment:
Suggest using %x instead of %u everywhere to save length.

Reply:
All %u instances replaced with %x